package.json

```diff
@@ -32,6 +32,7 @@
     "react": "^18",
     "react-dom": "^18",
     "react-hook-form": "^7.52.2",
+    "sharp": "^0.33.5",
```

Comment:
Why is the sharp dependency added?

Reply:
NextJS suggested it during build process of image optimization.